README.md

```diff
@@ -1,3 +1,150 @@
+
+<!--
+*** Thanks for checking out the Best-README-Template. If you have a suggestion
+*** that would make this better, please fork the repo and create a pull request
+*** or simply open an issue with the tag "enhancement".
+*** Thanks again! Now go create something AMAZING! :D
+***
+***
+***
+*** To avoid retyping too much info. Do a search and replace for the following:
+*** github_username, repo_name, twitter_handle, email, project_title, project_description
+-->
+
+
+
+<!-- PROJECT SHIELDS -->
+<!--
+*** I'm using markdown "reference style" links for readability.
+*** Reference links are enclosed in brackets [ ] instead of parentheses ( ).
+*** See the bottom of this document for the declaration of the reference variables
+*** for contributors-url, forks-url, etc. This is an optional, concise syntax you may use.
+*** https://www.markdownguide.org/basic-syntax/#reference-style-links
+-->
+
+
```

Comment:
The README was made with a template, wasn't it?
We can remove the extra text that we are not gonna use.

Reply:
hello, yes it was a template- I just copy pasted and left the text so everyone can see where I got it from, and also show the options of what can be added in the readme.. Im still getting to know the project, so did not want to be the only one changing and making decision about the readme.

Reply:
We could copy paste the text to our Notion into a subpage and keep it there for reference, hm?

Reply:
that sounds good!